packages/ui-react/src/components/VideoDetails/VideoDetails.module.scss

```diff
@@ -37,7 +37,7 @@
 
 .info {
   position: relative;
-  width: 70%;
+  width: 50%;
```

Comment:
I think we should remove `@include responsive.tablet-only() { width: 350px; }` couple of lines further. Because it makes the text description really narrow. 

![Screenshot 2024-01-29 at 12 51 06](https://github.com/Videodock/ott-web-app/assets/1019129/ec82c438-1721-4961-9afc-c062d4c54905)

So you agree @royschut ? If so, can you make perform this change.

Reply:
Wouldn't that break the purpose again, making text appear above the image?

Reply:
That is true. I think due to the length of the text it gives a really unproportionate feeling. But with a small text it feels more balanced. I have to agree with you. Example with small text:
![Uploading Screenshot 2024-01-30 at 09.31.33.png…]()

Reply:
I think we should actually show less of the image and more of the text. But that's a design (UI) choice.